app/src/main/java/com/nutomic/syncthingandroid/activities/SettingsActivity.java

```diff
@@ -363,6 +373,17 @@ private void setPreferenceCategoryChangeListener(
 
         public boolean onRunConditionPreferenceChange(Preference preference, Object o) {
             switch (preference.getKey()) {
+                case Constants.PREF_RUN_CONDITIONS:
+                    boolean enabled = (Boolean) o;
+                    for (int index = 1; index < mCategoryRunConditions.getPreferenceCount(); ++index) {
+                        mCategoryRunConditions.getPreference(index).setEnabled(enabled);
+                        // Log.d(TAG, "setEnabled");
```

Comment:
Forgotten debug log?

Reply:
Yes, thanks. Removed that line. In my tests it works fine. Of course I'm a bit wary to actually have my amateurish code in such a widely used project without further testing by other people. So if anyone else wants to give this a spin they would make me very happy..

Reply:
There's going to be a testing phase when the RC is released to the play beta channel. This seems safe enough (hey, the mainly active maintainer here isn't exactly an android specialist either ;) ).